### PR TITLE
Rebalances Marine turrets

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Changes up sentry stats, that's it.

Makes all sentries have a range of 7 because offscreen range is utterly silly for a sentry.

Reduced falloff to 0.05 for all sentries.

571 UAC- 30/20/0.5 and 250 rounds. 0.2 fire rate and 400 health
-
Mini Sentry- 22.5/30/1, 300 rounds, 0.15 fire rate and 600 health.
-
Old stats were 50/5/0 and 50 rounds/ 0.7 fire rate, higher health for UAC
25/5/0/0.3 for mini sentry.

## Why It's Good For The Game
Sentries are in a poor state because they feel more like snipers than actual sentries, this changes them to be more of an automatic support weapon rather than a sniper rifle that you stack 5 of and instagib a xeno because it fires so damn slowly.

## Changelog
:cl:
balance: Sentry faloff massively reduced. 571 UAC has 250 rounds and 25/15/1.25 and 0.2 fire rate from 0.7. Mini Sentry has 15/30/1.5 and 300 rounds. Sentry health upped to 600 for mini and 400 for UAC. Old stats were 50/5/0 and 0.7 fire rate for big and 25/5/0/0.3 for mini sentry.
/:cl:

### CODE REVIEW

**Review by hyper2snyper — APPROVED** [review]
Ima slap an approve sticker on this just incase my approval has any sway.
I will gladly admit this is far better than what I balanced before.

**Comment by Pariah919** [comment]
changed it to 30/20/0.5 from 25/15/1 and 22.5/30/1 from 15/30/1.5

**Review by SplinterGP — APPROVED** [review]
okayish

File: code/modules/projectiles/guns/sentries.dm [hunk 1]
```diff
@@ -62,9 +62,10 @@
 	desc = "A deployable, semi-automated turret with AI targeting capabilities. Armed with a M30 autocannon and a 25-round box magazine. Use Alt-Click to remove its battery."
 	icon_state = "sentry"
 
-	turret_range = 8
 	deploy_time = 8 SECONDS
 	max_shells = 50
+	fire_delay = 0.2 SECONDS
```
Comment (BraveMole) [inline code/modules/projectiles/guns/sentries.dm:67]: You upped the dps by 1.8 overall, not counting the pene, the sunder, the damage falloff, and the higer projectiles count

This is way too much, sentries are not that bad.

File: code/modules/projectiles/ammo_datums.dm [hunk 2]
```diff
@@ -1048,9 +1049,9 @@ datum/ammo/bullet/revolver/tp44
 
 /datum/ammo/bullet/turret/mini
 	name = "small caliber autocannon bullet"
-	damage = 25
-	penetration = 5
-	flags_ammo_behavior = AMMO_BALLISTIC|AMMO_SENTRY
+	damage = 22.5
+	penetration = 30
```
Comment (BraveMole) [inline code/modules/projectiles/ammo_datums.dm:1053]: this penetration is bwoinkers

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/ammo_datums.dm (hunk 2 above)
Comment (Pariah919) [inline code/modules/projectiles/ammo_datums.dm:1053]:
```suggestion
	penetration = 20
```

File: code/modules/projectiles/guns/sentries.dm (hunk 1 above)
Comment (SplinterGP) [reply code/modules/projectiles/guns/sentries.dm:67]: yes they were, at some tiles you'd take 8 seconds to kill a single drone standing still.

**Comment by David-Stormwell** [comment]
May be way powercreeped, tests are needed to make sure it's balanced enough.

**Comment by MiniMeatwad** [comment]
Very high health and requires a boiler to be able to destroy it since it can't be spit at anymore.

**Review by BraveMole — CHANGES_REQUESTED** [review]
Reduce the dps. Also mini sentry is stronger than UAC despite being half the price?

**Comment by Pariah919** [comment]
will reopen when i have better ideas

**Pull request closed without merging**